### ISSUE TITLE
Package a monospace font

### ISSUE THREAD

**Issue by garrettjstevens** [GMOD/jbrowse-components#808]
Currently we package the Roboto font so that JBrowse users on any platform will have a consistent experience (Roboto was chosen because it is the [suggested font for Material-UI](https://material-ui.com/getting-started/installation/#roboto-font)).

We don't package a monospace font, though, and rely on the user's system fonts (or generic font fallbacks), meaning things like the callback editors and pileup renderer may vary depending on the user's system.

I propose we package a monospace font so that users have a consistent experience. I suggest the [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono) font, for consistency with the font we already have.

Biggest downside I can see is increasing our bundle size a bit.

**Comment by rbuels** [comment]
I assume this is prompted right now by the recent test failures on Mac v
Linux?

**Comment by garrettjstevens** [comment]
Yeah, that's what made me think of it and I figured I'd put it in an issue before I forgot.

**Comment by cmdcolin** [comment]
might close for now...the basic monospace should be good enough
